earthpy/data/README.md

```diff
@@ -0,0 +1,180 @@
+data. It is strongly recommended that these data are directly acquired from an
+NPS server and not indirectly through other sources which may have changed the
+data in some way. Although these data have been processed successfully on
+computer systems at the National Park Service, no warranty expressed or implied
+is made regarding the utility of the data on other systems for general or
+scientific purposes, nor shall the act of distribution constitute any such
+warranty. This disclaimer applies both to individual use of the data and
+aggregate use with other data.
+
+
+
+# Rocky Mountain National Park digital elevation model
+
+A digital elevation model for Rocky Mountain National Park.
+
+### Filename
+
+`rmnp-dem.tif`
+
+### Source
+
+Created from terrain tiles on Amazon Web Services:
+https://aws.amazon.com/public-datasets/terrain/
+
+### License
+
+License information can be found here: https://mapzen.com/terms/
+
+
+
+# Colorado glacier locations
+
+Point locations of glaciers in the state of Colorado.
+
+### Filename
+
+`colorado-glaciers.geojson`
+
+### Source
+
+This dataset is a subset of the GLIMS Glacier Database that includes
+point locations of glaciers in the state of Colorado, USA, with the original
+source data available here:
+http://www.glims.org/download/glims_db_20171027.zip
+
+### License
+
+License information and other documentation can be found at
+http://www.glims.org/download/00README_GLIMS.txt
+
+
+
+# RGB imagery for Rocky Mountain National Park
+
+Low resolution RGB satellite imagery over RMNP, as a three channel GeoTIFF, and
+separate one channel GeoTIFF files.
+
+### Filename
+
+`rmnp-rgb.tif`, `red.tif`, `green.tif`, `blue.tif`
+
+### Source
+
+This Landsat imagery was originally showcased on NASA's Visible Earth website:
+https://visibleearth.nasa.gov/view.php?id=88405.
+The data provided in Earthpy have been spatially coarsened and reprojected to
+reduce the file size, and we have also provided additional single band GeoTIFF
+files (`red.tif`, `green.tif`, and `blue.tif`), to use in examples that stack
+raster layers.
+
+### License
+
+There are no restrictions on the use of data received from the U.S. Geological
+Survey's Earth Resources Observation and Science (EROS) Center or NASA's Land
+Processes Distributed Active Archive Center (LP DAAC), unless expressly
+identified prior to or at the time of receipt. More information on licensing
+and Landsat data citation is available from USGS.
+
+
+
+# Colorado counties polygons
+
+Simplified polygons for every county in Colorado.
+
+### Filename
+
+`colorado-counties.geojson`
+
+### Source
+
+U.S. Census Bureau: https://www.census.gov/geo/maps-data/data/cbf/cbf_counties.html
+
+### License
+
+Public Domain
+
+
+
+# EPSG code to proj4string dictionary
+
+A dictionary that maps numeric EPSG codes to their proj4string representations.
+
+### Filename
+
+`epsg.json`
+
+### Source
+
+TODO: name source
```

Comment:
Not sure where this data was acquired, but we can put the source here

Reply:
Should this go in our docs too? i suppose we can add it as an include especially if we use rst.

Reply:
EPSG data came from here via Chris H. :
```
# This EPSG mapping converted from:
# https://github.com/jswhit/pyproj/blob/master/lib/pyproj/data/epsg
```

Reply:
Thanks! Added this in 944b8b37f88ee8b0abc3719ab1efbb37bc05be9c